modules/asset/ui/src/index.js

```diff
@@ -1,3 +1,16 @@
 export default () => {
-  console.log('look at me, I am project level js');
+  const localesSelectorElem = document.querySelector('[data-locales-selector]');
+  const toggleBtn = localesSelectorElem.querySelector('[data-locales-toggle]');
+  const localesList = localesSelectorElem.querySelector('.locales__list');
+
+  window.addEventListener('click', ({ target }) => {
```

Comment:
I don't understand why this javascript exists. Isn't it just a link to the document's URL in the other locale?

Reply:
Well I needed a click event to open the locales dropdown. I can replace it with a simple hover if you prefer.

Reply:
OK understood.